xerosdk/apis/contacts.py

```diff
@@ -35,3 +36,15 @@ def post(self, data):
         """
 
         return self._post_request(data, Contacts.POST_CONTACTS)
+
+
+    def search_contact_by_display_name(self,display_name=None):
```

Comment:
`self, display_name: str`
display_name should be mandatory

Reply:
okay

Reply:
Why are we calling it display name and not contact name? display name is a QBO term right?

Reply:
okay replacing display_name with contact_name